lib/linguist/languages.yml

```diff
@@ -2905,6 +2905,19 @@ NewLisp:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 247
+Nextflow:
+  type: programming
+  ace_mode: groovy
+  tm_scope: source.nextflow
+  color: "#3ac486"
+  extensions:
+  - ".nf"
+  filenames:
+  - "main.nf"
```

Comment:
This is already included in `.nf`. Any reason to add it?

Reply:
if `filenames` doesn't have any special semantic no.

Reply:
`filenames` is just a list of... well, filenames to recognize as the given language. For example, it includes `.bashrc` for Shell. I think you can remove it.

Reply:
sure thing. done.